salt/elife-dashboard/config/srv-app-dashboard-app.cfg

```diff
@@ -0,0 +1,31 @@
+[article_scheduler]
+article_scheduler_url = http://localhost:8001/schedule/v1/article_scheduled_status/
+article_scheduler_publication_url = http://localhost:8001/schedule/v1/schedule_article_publication/
+article_scheduler_range_url = http://localhost:8001/schedule/v1/article_schedule_for_range/from/<from>/to/<to>/
+
+[django]
+debug = true
+allowed_hosts =
+secret_key = {{ pillar.elife_dashboard_2.secret_key }}
+
+[database]
+name: {{ pillar.elife_dashboard.db.name }}
+engine: django.db.backends.postgresql
+host = {{ pillar.elife_dashboard.db.host }}
+password = {{ pillar.elife_dashboard.db.password }}
+port = {{ pillar.elife_dashboard.db.port }}
+user =
+
+[elife]
+preview_base_url = {{ pillar.elife_dashboard_2.preview_base_url }}
+
+[logging]
+log_level =
+log_file =
+
+[sqs]
```

Comment:
This is because it's not used yet? A bit dangerous to leave fake values there, better to comment them out if not read?

Reply:
OK, I have commented them out. The `settings.py` handles them not being in the `app.cfg` so it's fine.